documentation/source/conf.py

```diff
@@ -169,6 +169,8 @@
     ("py:class", "jaxtyping.Shaped[Array, 'n']"),
     ("py:class", "jaxtyping.Shaped[ndarray, 'n']"),
     ("py:class", "jax._src.typing.SupportsDType"),
+    ("py:class", "numpy.bool"),
```

Comment:
I don't think this line is required. I've added the next one as part of #675. Maybe safest to delete both of these from this PR to pre-empty junk and/or merge conflicts.